src/main/webapp/app/components/pipes/sort-by.pipe.ts

```diff
@@ -53,6 +53,7 @@ export class SortByPipe implements PipeTransform {
         return quizExercise.isVisibleBeforeStart ? 3 : 4;
     }
 
+    // TODO: How does this work? An exercise does not have a completion date.
     durationForExercise(exercise: any) {
```

Comment:
```ts
    durationForExercise(exercise: Exercise) { .. }
```
We should properly type this method! Also, what is the return type. (I guess this is out of scope for this PR, but just wondering.)

Reply:
I guess this is simply wrong and we should fix it. I assume that it rather is a result instead of an exercise. I guess we have to rework the whole pipe to fix such issues. And this is out of scope for the PR...